### ISSUE TITLE
dump release script: restrict to complete and/or reviewed content

### ISSUE THREAD

**Issue by lgeistlinger** [waldronlab/BugSigDBExports#7]
incorporate filter for completion and/or review status in dump_release script as available from https://github.com/waldronlab/BugSigDB/issues/92, in order to promote https://github.com/waldronlab/BugSigDBExports/issues/4

**Comment by lgeistlinger** [comment]
Hi @lwaldron,

A quick look at the current "complete"  and "reviewed" status:  

1. "Complete":

```
> df <- bugsigdbr::importBugSigDB()
> table(df[["State"]])

  Complete Incomplete 
      2022         20 

> sum(df[["State"]] == "Complete") / nrow(df)
[1] 0.9524258
```

2. "Reviewed":

```
> table(df[["Reviewer"]])

            Chloe            Fatima      Fatima Zohra      LGeistlinger 
               10                82               113                 2 
      Lucy Mellor          Lwaldron      Rimsha Azhar Shaimaa Elsafoury 
                2                15                22               391 
> sum(table(df[["Reviewer"]])) / nrow(df)
[1] 0.3000471
```

It looks like reviewing signatures will be a bigger task, so let's maybe go ahead with all complete signatures for our BugSigDB 1.0 release?

**Comment by lwaldron** [comment]
Sounds good to me. These signatures are all by curators who received training, so they are already by default more QCed than contributions from the world at large.

**Comment by lgeistlinger** [comment]
closed via 21e58cd
